atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/BackgroundSweeperImpl.java

```diff
@@ -42,6 +44,14 @@
 public final class BackgroundSweeperImpl implements BackgroundSweeper {
     private static final Logger log = LoggerFactory.getLogger(BackgroundSweeperImpl.class);
 
+    // We've noticed that Sweep increases GC pressure in cassandra when sweeping the VALUE table of a StreamStore.
+    // Thus, we use a reduced config in these scenarios.
+    private static final ImmutableSweepBatchConfig STREAM_STORE_BATCH_CONFIG = ImmutableSweepBatchConfig.builder()
+            .candidateBatchSize(128)
```

Comment:
I thought about using `config.batchSize / 8`, but since we have a good idea of the size of the data we'll encounter (for once), I think these are fine.